numpy/core/tests/test_arrayprint.py

```diff
@@ -237,12 +237,27 @@ def test_formatter_reset(self):
         assert_equal(repr(x), "array([0., 1., 2.])")
 
     def test_0d_arrays(self):
+        if sys.version_info[0] >= 3:
+            assert_equal(str(np.array('café', np.unicode_)), 'café')
+            assert_equal(repr(np.array('café', np.unicode_)),
+                         "array('café',\n      dtype='<U4')")
```

Comment:
This has an issue number, right?

Reply:
I don't think so... I think we came up with this one in #9143.